tests/conftest.py

```diff
@@ -285,6 +285,12 @@ def es(app):
         current_search_client.indices.delete_template('*')
 
 
+@pytest.yield_fixture()
+def index_prefix(app):
+    """Add index prefix to the app config."""
+    app.config.update(dict(SEARCH_INDEX_PREFIX='test-'))
```

Comment:
I think this will modify the configuration permanently. The common practice is something like:

```python
@pytest.fixture
def config_with_index_prefix(app):
    # set the config (store the original value as well just to be sure)
    original_value = app.config.get('SEARCH_INDEX_PREFIX')
    app.config['SEARCH_INDEX_PREFIX'] = 'test-'

    # yield it
    yield app.confg

    # set the original value back
    app.config['SEARCH_INDEX_PREFIX'] = original_value
```